lib/bootstrap/patches/jar_dependencies.rb

```diff
@@ -21,11 +21,33 @@ def require_jar(*args)
   return nil unless Jars.require?
   result = Jars.require_jar(*args)
   if result.is_a? String
-    # JAR_DEBUG=1 will now show theses
+    # JARS_VERBOSE=true will show these
     Jars.debug { "--- jar coordinate #{args[0..-2].join(':')} already loaded with version #{result} - omit version #{args[-1]}" }
     Jars.debug { "    try to load from #{caller.join("\n\t")}" }
     return false
   end
   Jars.debug { "    register #{args.inspect} - #{result == true}" }
   result
 end
+
```

Comment:
the backport is missing comments https://github.com/elastic/logstash/pull/16919/files#diff-799a9c7c14d2dace2d5e32653020eeb792ab3b161e6cd4a734226b87086a6e4aR33-R36